src/System.Text.Json/tests/RealWorld/BlogPost.cs

```diff
@@ -0,0 +1,850 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace System.Text.Json.Tests.RealWorld.BlogPost
+{
+    public class Test : RealWorldTest
+    {
+        public override void Initialize()
+        {
+            // This payload was obtained from https://medium.com/@fagnerbrack/a-smart-programmer-understands-the-problems-worth-fixing-dcf15871f943?format=json.
+            PayloadAsString = File.ReadAllText("Resources/Payloads/BlogPost.json").StripWhitespace();
```

Comment:
What's the license associated with it?  Are we able to just incorporate it into the repo?  Can we instead create an example that has similar characteristics but that's not a copy of code from elsewhere?

Reply:
Or perhaps replace all the strings with garbage.